cg/cli/store/fastq.py

```diff
@@ -1,84 +1,131 @@
+from housekeeper.store.models import File
+
+from cg.apps.crunchy.files import update_metadata_paths
 from cg.cli.compress.helpers import get_fastq_individuals, update_compress_api
+from cg.constants import SequencingFileTag
+from cg.constants.constants import DRY_RUN
 from cg.exc import CaseNotFoundError
 from cg.meta.compress import CompressAPI
 from cg.models.cg_config import CGConfig
-from cg.store import Store, models
+from cg.store import Store
+from cg.store.models import Sample
 
 LOG = logging.getLogger(__name__)
 
 
 @click.command("sample")
 @click.argument("sample-id", type=str)
-@click.option("-d", "--dry-run", is_flag=True)
+@DRY_RUN
 @click.pass_obj
-def store_sample(context: CGConfig, sample_id: str, dry_run: bool):
-    """Include links to decompressed FASTQ files belonging to this sample in housekeeper"""
+def store_sample(context: CGConfig, sample_id: str, dry_run: bool) -> int:
+    """Include links to decompressed FASTQ files belonging to this sample in Housekeeper."""
     compress_api: CompressAPI = context.meta_apis["compress_api"]
     status_db: Store = context.status_db
     update_compress_api(compress_api, dry_run=dry_run)
-
-    sample_obj = status_db.sample(internal_id=sample_id)
-    was_decompressed: bool = compress_api.add_decompressed_fastq(sample_obj)
+    sample = status_db.sample(internal_id=sample_id)
+    if not sample:
+        LOG.warning(f"Could not find {sample_id}")
+        return 0
+    was_decompressed: bool = compress_api.add_decompressed_fastq(sample)
     if not was_decompressed:
         LOG.info(f"Skipping sample {sample_id}")
         return 0
     LOG.info(f"Stored fastq files for {sample_id}")
     return 1
 
 
+def invoke_store_samples(context: click.Context, sample_ids: List[str], dry_run: bool) -> int:
+    """Invoke store sample for sample ids."""
+    stored_individuals: int = 0
+    for sample_id in sample_ids:
+        stored_count: int = context.invoke(store_sample, sample_id=sample_id, dry_run=dry_run)
+        stored_individuals += stored_count
+    return stored_individuals
+
+
 @click.command("case")
 @click.argument("case-id", type=str)
-@click.option("-d", "--dry-run", is_flag=True)
+@DRY_RUN
 @click.pass_context
-def store_case(context: click.Context, case_id: str, dry_run: bool):
-    """Include links to decompressed FASTQ files belonging to this case in housekeeper"""
+def store_case(context: click.Context, case_id: str, dry_run: bool) -> None:
+    """Include links to decompressed FASTQ files belonging to this case in Housekeeper."""
 
     status_db: Store = context.obj.status_db
     try:
-        samples: Iterable[str] = get_fastq_individuals(status_db, case_id)
-        stored_individuals = 0
-        for sample_id in samples:
-            stored_count: int = context.invoke(store_sample, sample_id=sample_id, dry_run=dry_run)
-            stored_individuals += stored_count
+        samples: Iterable[str] = get_fastq_individuals(store=status_db, case_id=case_id)
```

Comment:
Not sure why the `get_fastq_individuals` function exists, I think it can just be replaced by the status_db method
```suggestion
        samples: Iterable[str] = status_db.get_samples_by_family_id(family_id=case_id)
```